protoc-gen-openapiv2/internal/genopenapi/types.go

```diff
@@ -314,6 +314,8 @@ type openapiSchemaObject struct {
 	MaxProperties    uint64   `json:"maxProperties,omitempty" yaml:"maxProperties,omitempty"`
 	MinProperties    uint64   `json:"minProperties,omitempty" yaml:"minProperties,omitempty"`
 	Required         []string `json:"required,omitempty" yaml:"required,omitempty"`
+
+	PathParamName string `json:"-" yaml:"-"`
```

Comment:
Should we add a `FieldConfiguration` struct here too to encapsulate potential future additions?

Reply:
Done! Also reran the code generation, as expected this did not generate any more changes.

Reply:
Looks like there are some failing tests 😬

Reply:
Fixed, but I'm not sure if this is the right way to fix it (see comment below).